### PR TITLE
Support setting update action body in config

### PR DESCRIPTION
**Pull request by Air-Miha**
I added a test for this case both for update and create, where it was missing.

### CODE REVIEW

File: src/actions/update.js [hunk 1]
```diff
@@ -30,19 +30,26 @@ import { extendMetaWithResponse } from './_rsaa';
...
 
+  let body = null;
+  if (item !== null) {
+    if (!_.isObject(item)) {
+      throw new Error('Item is not valid in method argument');
+    }
+    body = JSON.stringify({ data: item });
+  } else {
+    body = rsaaConfig.body;
+  }
```
Comment (vedrani) [inline src/actions/update.js]: I cannot comment line 72 ln. But that line si not correct. Does it make sense to put body in there? Or nothing? We should dicsuss it.

File: test/actions/create.spec.js [hunk 2]
```diff
@@ -312,6 +312,26 @@ describe('Create action creator', () => {
       .to.not.throw('Item is missing in method argument and in config.body');
```
Comment (vedrani) [inline test/actions/create.spec.js]: Is this obsolete, I cannot find this exception, do you?

File: test/actions/update.spec.js [hunk 3]
```diff
@@ -175,7 +175,44 @@ describe('Update action creator', () => {
...
+      endpoint: 'api.test',
+    };
+
+    const schema = 'app.builder';
+    const schemaConfig = {
+      schema,
+      request: config,
+    };
+    expect(() => update(schemaConfig))
+      .to.not.throw('Item is missing in method argument and in config.body');
```
Comment (vedrani) [inline test/actions/update.spec.js]: Cannot find who can throw this

File: test/actions/update.spec.js [hunk 4]
```diff
@@ -175,7 +175,44 @@ describe('Update action creator', () => {
...
+
+    const schema = 'app.builder';
+    const schemaConfig = {
+      schema,
+      request: config,
+    };
+
+    const action = update(schemaConfig);
+
+    expect(action[RSAA].body).to.equal(config.body);
```
Comment (vedrani) [inline test/actions/update.spec.js]: Add test that tests what if I pass item and body. Do it please for create and update.

File: src/actions/update.js (hunk 1 above)
Comment (Air-Miha) [inline src/actions/update.js]: You're right, we should remove it. I totally missed this line.

File: test/actions/create.spec.js (hunk 2 above)
Comment (Air-Miha) [inline test/actions/create.spec.js]: I thought that it was somewhere in other parts of the code. I tried searching for anything that would resemble this exception but I didn't find anything. You're right, it seems to be obsolete.

File: test/actions/update.spec.js (hunk 4 above)
Comment (Air-Miha) [inline test/actions/update.spec.js]: There is already such a test for create. I added one for update.

File: src/actions/update.js (hunk 1 above)
Comment (vedrani) [inline src/actions/update.js]: I taught we are gona discuss it first. RIO is optimistic with updates in state, I would like to keep it.

**Comment by Air-Miha** [comment]
Sorry about that. I misunderstood the comment about the discussion. I've reverted the commit. Let's discuss this. We have a concrete use case for this in the auth extension.

**Pull request merged**